### PR TITLE
Add sequence mode for audio

### PR DESCRIPTION
**Pull request by mzlarsson**
Makes it possible to specify multiple assets and we will play them directly after one another, as a single action. It is also possible to give the `sequenceLooping` argument to specify the number of loops for each clip. This allows you to for example have one clip, then a looping paus (now-are-the-characters-talking) clip in the middle and then another clip without explicitly timing the start of each clip. This could allow for smoother transitions between songs that need to be split in looping parts.

### CODE REVIEW

**Comment by AntonNeld** [comment]
It makes sense to have this functionality regardless (except maybe the number of loops), but this is about Hackermusik II, right? Don't we want to trigger going from the loop to the next part of the song manually, instead of looping it a fixed number of times?

**Comment by mzlarsson** [comment]
> It makes sense to have this functionality regardless (except maybe the number of loops), but this is about Hackermusik II, right? Don't we want to trigger going from the loop to the next part of the song manually, instead of looping it a fixed number of times?

Yeah it is. My thought was that you set the looping to be infinite and then we can just toggle looping to OFF in the UI to move on to the next part.

File: media/model/media/audiohandler.js [hunk 1]
```diff
@@ -84,6 +102,10 @@ module.exports = class AudioHandler extends MediaHandler {
                 this.stopFade();
                 this.setMuted(!this.isMuted());
                 break;
+            case 'toggle_loop':
```
Comment (mzlarsson) [inline media/model/media/audiohandler.js:105]: This part (and `toggleLoop` below) is required for Ragnarök. Should I do a separate PR with just these parts perhaps?

**Review by AntonNeld — APPROVED** [review]

**Comment by mzlarsson** [comment]
We found another solution and made toggleLoop in its own PR.
